cypress/e2e/features/backend/aa_upgrade_downgrade_usersubscription.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Upgrade or downgrade usersubscription
+
+    Background: User gets refresh_token
+        Given User should get refresh_token and create authToken
+   
+    Scenario: Upgrade or downgrade usersubscription to any tier
+        Given User should be able to upgrade or downgrade usersubscription to "tier_1"
+        
```

Comment:
It s good to keep steps separate. You have already written Given in the background. You can say like When I send request to.... Then I verify...

Reply:
done

Reply:
What will happen If tier type is tier 2 for other user? Let s think about make this scenario even more dynamic. You already check tier with switch statement in the step definition. So what happens if you send all the possible values from data table here (like tier 1, tier2,..) and get the tier type from the user_subscription, pass it to the switch statement and it will make the assertion according to tier plan.

Reply:
done